src/tabs/pid_tuning.html

```diff
@@ -21,9 +21,12 @@
                 <div class="head" i18n="pidTuningRateProfile"></div>
                 <div class="bottomarea">
                     <select name="rate_profile">
-                        <option value="0">Rateprofile 1</option>
-                        <option value="1">Rateprofile 2</option>
-                        <option value="2">Rateprofile 3</option>
+                        <option value="0" class="RateProfile1">Rateprofile 1</option>
```

Comment:
@McGiverGim: Looking at this, shouldn't these be localised as well?

Reply:
Yes, if it can be localised here, perfect. If not I will need to do a PR after that to make it, so better here.

Reply:
If you want to let the list of options as static values in the html, only need is an `i18n` attribute in the option tag. If make programmatically in the `.js` files, then the `i18n.getMessage()` function must be called.

Reply:
As an alternative, I was considering changing the options to be numbers only. If not, then doing it programmatically will avoid having to maintain the same expression 6 times.

Will merge this for now since it's tied to a Betaflight pull request that was merged, can solve the i18n in another pull request.

Reply:
And maybe is cleaner, in the `.js` to hide the `option` by `value` and not adding a different `class` to each of them.

Reply:
That was copy / pasta from another place where we are doing this...

Reply:
@McGiverGim: If you are externalising another round of strings for i18n, please pick these up as well - I don't know when I'll have time to fix this up.